mcf-browser/src/back-end/utils.ts

```diff
@@ -33,6 +33,7 @@ const ERROR_MESSAGES =
       'assert-noCur': 'current node must be set before declaring properties',
       'parse-noColon': 'missing \':\', incorrect mcf triple format',
       'parse-noLabel': 'missing property label',
+      'multiple-tmcf': 'multiple TMCF files were uploaded, only the last one was used',
```

Comment:
nit: we want constants to be in all caps and with underscores, could we update all of these

eg. `'multiple-tmcf'` => MULTIPLE_TMCF & to use it, we would do: ERROR_MESSAGES.MULTIPLE_TMCF

Reply:
Changed!